src/components/Recorder/index.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jsx-a11y/no-noninteractive-tabindex */
```

Comment:
Add disable comment to only certain lines, rather than the whole file. Also, be doubly sure that tabIndex is required in those elements.